imports/api/files/files.js

```diff
@@ -1,21 +1,14 @@
 import { Mongo } from 'meteor/mongo';
 
+if (Meteor.isServer) {
```

Comment:
instead of splitting to server/files and client/files you could use a single file with Meteor.isServer condition. It looks like the only difference between the client and server is in `new FS.Store.S3('files', {...})`

Reply:
@pgorecki using `Meteor.isServer` considered insecure
(see https://github.com/CollectionFS/Meteor-CollectionFS/tree/devel/packages/s3)
> As given in the above code but located in a directory named server (note: wrapping in Meteor.isServer is NOT secure).

Reply:
I don't see any security issues here, 
```
const filesStore = new FS.Store.S3('files', {
 +  chunkSize: 512 * 1024,
 +  accessKeyId: Meteor.settings.s3config.key,
 +  secretAccessKey: Meteor.settings.s3config.secret,
 +  bucket: Meteor.settings.s3config.bucket,
 +});
```
we are keeping all secretes in Meteor.settings which are not available on client side anyway